cmd/describe.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/redhat-developer/ocdev/pkg/application"
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/redhat-developer/ocdev/pkg/occlient"
+	"github.com/redhat-developer/ocdev/pkg/project"
+	"github.com/redhat-developer/ocdev/pkg/storage"
+	"github.com/redhat-developer/ocdev/pkg/url"
+	"github.com/spf13/cobra"
+)
+
+var componentDescribeCmd = &cobra.Command{
```

Comment:
this should be called `describeCmd`

Reply:
Since `create.go`, `delete.go` followed the same the pattern, I did the same But I will change it (& then will change other in different PR)